pgrx-sql-entity-graph/src/extension_sql/entity.rs

```diff
@@ -134,13 +134,13 @@ impl Display for SqlDeclaredEntity {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
             SqlDeclaredEntity::Type(data) => {
-                f.write_str(&(String::from("Type(") + &data.name + ")"))
+                write!(f, "Type({})", data.name)
             }
             SqlDeclaredEntity::Enum(data) => {
-                f.write_str(&(String::from("Enum(") + &data.name + ")"))
+                write!(f, "Enum({})", data.name)
             }
             SqlDeclaredEntity::Function(data) => {
-                f.write_str(&(String::from("Function ") + &data.name + ")"))
+                write!(f, "Function({})", data.name)
```

Comment:
I can't actually find a case where this actually diffs a schema, and that is Worrying, because that suggests it is Simply Dead Code.

Reply:
I did not exhaustively cover every extension in reach but I have reached the "I need to script out the schema generation delta check" part, which means I might as well bash out a v0.1 solution to https://github.com/pgcentralfoundation/pgrx/pull/1521